docs/taskruns.md

```diff
@@ -606,6 +657,8 @@ spec:
 
 An example TaskRun definition could look like:
 
+{{< tabs >}}
```

Comment:
Suggest renaming this section to "Configuring Task Steps and Sidecars in a TaskRun" and updating references to "StepOverrides" and "SidecarOverrides"